robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -63,7 +64,6 @@ Create Relationships for contacts
     # Load Related List                   Relationships
```

Comment:
As mentioned earlier this is a comment that could be removed.

Reply:
done